ocaml/xapi/xapi_sdn_controller.ml

```diff
@@ -0,0 +1,54 @@
+(*
+ * Copyright (C) 2006-2016 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Stdext
+open Pervasiveext
+open Xstringext
+open Helpers
```

Comment:
We don't see to need most of these `open`s, actually. Please only open what is really required. Looks like only `Network` is used.